tests/untl_structure_test.py

```diff
@@ -587,3 +587,59 @@ def test_generate_form_data(_):
     assert isinstance(fg, us.FormGenerator)
     # Check missing children were added.
     assert len(metadata.children) == len(metadata.contained_children)
+
+
+@pytest.mark.parametrize('test_input_content, test_output',
+                         [
+                             ([us.Meta(content='DC', qualifier='system'),
+                               us.Meta(content='True', qualifier='hidden')],
+                              True),
+                             ([us.Meta(content='DC', qualifier='system'),
+                               us.Meta(content='False', qualifier='hidden')],
+                              False),
+                         ])
+def test_Metadata_is_hidden(test_input_content, test_output):
+    """Check if a UNTL element is hidden."""
+    metadata = us.Metadata()
+    metadata.children = test_input_content
+    assert metadata.is_hidden is test_output
+
+
+def test_Metadata_is_hidden_with_no_meta_hidden_element(capsys):
+    metadata = us.Metadata()
+    metadata.children = [us.Meta(content='DC', qualifier='system')]
+    assert metadata.is_hidden is None
+    captured = capsys.readouterr()
+    assert captured.err == 'A hidden meta element does not exist.'
+
+
+@pytest.mark.parametrize('test_input_elements, test_input_content, test_output',
+                         [
+                             ([us.Meta(content='DC', qualifier='system'),
+                               us.Meta(content='True', qualifier='hidden')],
+                              True, True),
+                             ([us.Meta(content='DC', qualifier='system'),
+                               us.Meta(content='False', qualifier='hidden')],
+                              False, True),
+                             ([us.Meta(content='DC', qualifier='system')],
+                              None, False)
+                         ])
+def test_Metadata_make_hidden(test_input_elements, test_input_content, test_output):
+    """Test if a UNTL unhidden element is altered to hidden."""
+    metadata = us.Metadata()
+    metadata.children = test_input_elements
+    assert metadata.is_hidden is test_input_content
+    metadata.make_hidden()
+    assert metadata.is_hidden is test_output
+
+
+@pytest.mark.parametrize('test_input_content, test_output', [('True', True), ('False', False)])
+def test_Metadata_make_unhidden(test_input_content, test_output):
+    """Test if a UNTL hidden element is altered to unhidden."""
+    metadata = us.Metadata()
+    system = us.Meta(content='DC', qualifier='system')
+    hidden = us.Meta(content=test_input_content, qualifier='hidden')
+    metadata.children = [system, hidden]
+    assert metadata.is_hidden is test_output
+    metadata.make_unhidden()
+    assert metadata.is_hidden is False
```

Comment:
The `test_Metadata_make_unhidden` test is written a bit differently than you did the `test_Metadata_make_hidden` with the way you did the `parametrize` arguments. I like the way you did the arguments in the `test_Metadata_make_hidden` because it is easier for me to read the intent and expectation of the test; would you mind writing the `test_Metadata_make_unhidden` test in a consistent style with that?